src/components/NurseryWithdrawals/PlantingProgressTabContent.tsx

```diff
@@ -91,12 +93,13 @@ export default function PlantingProgress({ reloadTracking }: PlantingProgressPro
         style={{ padding: isMobile ? theme.spacing(3) : theme.spacing(3, 0, 0) }}
         initialView={initialView}
         onView={setActiveView}
-        search={<SearchComponent view={activeView} onChangePlantingSite={setSelectedPlantingSiteId} {...searchProps} />}
+        search={<SearchComponent view={activeView} {...searchProps} />}
         list={
           <PlantingProgressList
             search={search}
             plantingCompleted={plantingCompleted}
             reloadTracking={reloadTrackingAndObservations}
+            plantingSiteId={selectedPlantingSiteId}
           />
         }
         map={<PlantingProgressMap plantingSiteId={selectedPlantingSiteId} />}
```

Comment:
we need some changes in the map as well, if site id is -1 there is a different messaging

Reply:
sounds good 👍